src/src_cpp/src_test/CMakeLists.txt

```diff
@@ -37,14 +40,9 @@ target_link_libraries (unit_tests PRIVATE
 		GTest::Main
 		helpers
 		lpnamer
+		${PROJECT_NAME}::sequentialcore
 	)
 
-if (USE_SEQUENTIAL)
-	target_link_libraries (unit_tests PRIVATE
-			${PROJECT_NAME}::sequentialcore
-	)
-endif ()
-
```

Comment:
`USE_SEQUENTIAL` could still be defined to `OFF `or `USE_SEQUENTIAL` is not needed anymore.
Need to keep link to `sequentialcore` only if used and needed.